jekyll.gemspec

```diff
@@ -36,7 +36,8 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency("i18n",                  "~> 0.7")
   s.add_runtime_dependency("jekyll-sass-converter", "~> 1.0")
   s.add_runtime_dependency("jekyll-watch",          "~> 2.0")
-  s.add_runtime_dependency("kramdown",              "~> 1.14")
+  s.add_runtime_dependency("kramdown",              "~> 2.1")
```

Comment:
@ashmaroli I was hoping to allow kramdown v1 to continue to be used by this change to prevent a breaking change in Jekyll v3. Is there anything that we should consider when allowing this besides conditionally loading dependencies based on the version? We could remove the below kramdown-parser-gfm dependency to enable that.

Reply:
> Is there anything that we should consider when allowing this..

AFAICR the noteworthy breaking change was kramdown extracting many of its builtin components into separate gems.
For additional details, the post published at the time kramdown-2.0 released can be read at: https://kramdown.gettalong.org/news.html#kramdown-200-released



> We could remove the below kramdown-parser-gfm dependency to enable that.

:+1: